### DISCUSSION TITLE
Obtaining Subject Matter information

### DISCUSSION THREAD

**Discussion by jlchang** [dandi/helpdesk#136] [General]
For electrophysiology data, I'm interested in metadata on the brain region where the recording was performed. For some Patch-seq studies, the appropriate ontology term was provided as the `Subject Matter` of the Dandiset (examples:  [000008](https://dandiarchive.org/dandiset/000008/), [000020](https://dandiarchive.org/dandiset/000020/), [000035)](https://dandiarchive.org/dandiset/000035)). Unfortunately, some Dandisets do not provide a `Subject Matter` entry, but they tend to have `Subject Matter`-like keyword (examples: [000043](https://dandiarchive.org/dandiset/000043), [000630](https://dandiarchive.org/dandiset/000630), [000636](https://dandiarchive.org/dandiset/000636)). Uberon ontology terms are also sometimes listed in the [Version Instance json](https://api.dandiarchive.org/api/dandisets/000008/versions/0.211014.0809/) (in the `about` section).

Is there another Metadata or Manifest location where brain region metadata can be more consistently located? If no, is there a way to identify the appropriate keyword to transform into an ontology term in lieu of a `Subject Matter` entry?

If the recordings are from different brain regions, does the [Nested Asset List](https://api.dandiarchive.org/api/dandisets/000008/versions/0.211014.0809/assets/) or the [Dandiset Manifest](https://dandiarchive.s3.amazonaws.com/dandisets/000008/0.211014.0809/assets.yaml) (assets.yaml) have affordances for indicating per recording brain region information (where applicable).

I notice similar interest in other discussion threads ["users will surely want to search recordings made in a given brain region"](https://github.com/dandi/helpdesk/discussions/100) and ["a UI that allows one to query [...] region of the brain"](https://github.com/dandi/helpdesk/discussions/113) and would be interested in any information about existing or upcoming features that could facilitate such dataset discovery.

**Comment by satra** [comment]
indeed the asset data model allows adding subject matter to each asset as well. but it is dependent on each provider to have provided that. a future version of tools will help users populate some of this information or at least prompt the user to increase richness of metadata or highlight what may be missing. we are also working with the NWB team to make such information be validated at the level of the NWB file, from which we can extract it into the asset schema.

**Comment by jlchang** [comment]
Given DANDI's neurophysiology focus, is it reasonable to have the UBERON term for "brain" (UBERON_0000955) to use as a default for indexing the dataset's tissue? Or are there sufficient non-brain samples in DANDI that such a hypothetical default assignment would be inaccurate for >1% of the data?

**Reply by satra** [reply]
generally all dandisets have some brain component. however, some datasets though are based on cultured cells, although they are being used to generate neurophysiology signals.

**Reply by satra** [reply]
perhaps you can tell us what you are trying to do. we have some ongoing efforts on knowledgebase projects that will provide a different view of dandisets by the end of summer.
